cssselect/xpath.py

```diff
@@ -58,7 +58,7 @@ def add_condition(self, condition):
         if self.condition:
             self.condition = '%s and (%s)' % (self.condition, condition)
         else:
-            self.condition = condition
+            self.condition = '(%s)' % condition
```

Comment:
isn't in simpler to wrap conditions only when there is one already ?
like only doing
```
        if self.condition:
            self.condition = '(%s) and (%s)' % (self.condition, condition)
```
so that simple single conditions are kept as currently